templates/cloudflare-workers/server.ts

```diff
@@ -1,7 +1,52 @@
-import { createEventHandler } from "@remix-run/cloudflare-workers";
+import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
+import type { AppLoadContext } from "@remix-run/cloudflare";
+import { createRequestHandler, logDevReady } from "@remix-run/cloudflare";
 import * as build from "@remix-run/dev/server-build";
+const handleRemixRequest = createRequestHandler(build, process.env.NODE_ENV);
 
-addEventListener(
-  "fetch",
-  createEventHandler({ build, mode: process.env.NODE_ENV })
-);
+if (build.dev) {
+  logDevReady(build);
+}
+
+export default {
+  async fetch(
+    request: Request,
+    env: {
+      __STATIC_CONTENT: Fetcher;
+      __STATIC_CONTENT_MANIFEST: string;
```

Comment:
Using `__STATIC_CONTENT_MANIFEST` on the environment like this is a Miniflare / Wrangler Local bug. It will break when using Wrangler in live mode or in production. You have to import it [like so](https://github.com/cloudflare/kv-asset-handler#es-module-1), and add `serverDependenciesToBundle: [/^(?!(__STATIC_CONTENT_MANIFEST)$).*$/u]` to your `remix.config.js`.

Reply:
i tried using the `server.ts` from this template with the latest versions of remix and wrangler and found that it broke my static assets in production. so i tried reverting this from using `env.__STATIC_CONTENT_MANIFEST` to using 
```ts
import assetManifestAsString from '__STATIC_CONTENT_MANIFEST';
const assetManifest = JSON.parse(assetManifestAsString);
```
and passing that as `ASSET_MANIFEST` to `getAssetFromKV()`, and it seemed to fix the css files that were 404ing but my `/build/root-{hash}.js` and `/build/manifest-{hash}.js` still 404ed in production. so i reverted `server.ts` back to:
```ts
import { createEventHandler } from "@remix-run/cloudflare-workers";
import * as build from "@remix-run/dev/server-build";

addEventListener(
  "fetch",
  createEventHandler({ build, mode: process.env.NODE_ENV })
);
```
and production is fixed (but no HMR in dev, obviously).